packages/gatsby-cli/src/reporter/loggers/ipc/index.ts

```diff
@@ -0,0 +1,57 @@
+import { onLogAction } from "../../redux/index"
+import { ISetStatus, ActionsUnion } from "../../redux/types"
+import { Actions, LogLevels } from "../../constants"
+import stripAnsi from "strip-ansi"
+
+const isISetStatus = (action: ActionsUnion): action is ISetStatus =>
+  typeof action.payload === `string`
```

Comment:
we could make the naming of this a little more generic. Today it applies to that action, but it could be less bound to that in the future.

`function isStringPayload()...`